src/main/java/nl/aerius/sldgenerator/plugin/FileScanner.java

```diff
@@ -31,11 +31,15 @@
    * @param fileExtension extension used to filter files
    * @return files as list
    */
-  public static List<File> findFilesWithExtension(final File searchPath, final String fileExtension) throws IOException {
-      return Files.walk(searchPath.toPath())
+  public static List<File>
+
```

Comment:
Seems like a linebreak to much here? Or is this caused by spotless

Reply:
Doh. No I changed that to see if spotless fixed such code. But I missed it between all other changes. It apparently didn't format this...